source/install_scripted_offline_rhel7_rhel8.rst

```diff
@@ -1,103 +1,84 @@
-Scripted Installation
+Offline Scripted Installation (RHEL 7/8 and CentOS 7/8)
 ----------------------
 The scripted installation runs an installer that uses Ansible to install the command line tool, quipucords server image, and the database image. When you run the scripted installation, the server is installed and started as described in `Configuring and Starting Quipucords <install.html#config-and-start>`_. However, you can change some of the defaults used by the scripted installation with the `Installation Options <install.html#install-opts>`_.
 
 Obtaining the Installer
 ^^^^^^^^^^^^^^^^^^^^^^^
-1. Download the installer by entering the following command::
+1. Download & Install the quipucords installer rpm.
```

Comment:
Should we call this the qpc-tools rpm? Since the rpm will only be available in that repo now?

Reply:
yes